fe/fe-core/src/main/java/org/apache/doris/analysis/AlterTableStmt.java

```diff
@@ -77,7 +78,15 @@ public void analyze(Analyzer analyzer) throws UserException {
         if (ops == null || ops.isEmpty()) {
             ErrorReport.reportAnalysisException(ErrorCode.ERR_NO_ALTER_OPERATION);
         }
+        Table table = null;
+        if (!FeConstants.runningUnitTest) {
+            table = Env.getCurrentInternalCatalog().getDbOrDdlException(tbl.getDb())
+                    .getTableOrDdlException(tbl.getTbl());
+        }
         for (AlterClause op : ops) {
+            if (op instanceof AlterTableClause && table instanceof OlapTable) {
+                ((AlterTableClause) op).setOlapTable((OlapTable) table);
```

Comment:
it better to table name no OlapTable object

Reply:
Done